README.md

```diff
@@ -17,32 +17,34 @@ We use Docker for local development. To get started, run the following from your
     cd committee-oversight
     ```
 
-2. The root of this repository has a file named `hearings.dump`, a sample set of hearings data for use in local development. To load that into your database, run:
+2. DataMade employees can download the most recent copy of the scraped committee hearings `hearings.dump` from datamade-postgresql-backups/hearings in S3. Otherwise you'll need to run the scrapers in https://github.com/datamade/hearings/ to create you own hearings database.
```

Comment:
```suggestion
2. DataMade employees can download the most recent copy of the scraped committee hearings `hearings.dump` from `datamade-postgresql-backups/hearings` bucket in S3. Otherwise you'll need to run [the hearings scraper]( https://github.com/datamade/hearings/) to populate your application database with the latest committee hearings. **Subsequent steps will not work until you add initial hearings data to your database.**
```